### ISSUE TITLE
Support for Angular 2 Static Template Compilation

### ISSUE THREAD

**Issue by guillaumeleone** [FountainJS/generator-fountain-webapp#14]
Hello,

```
? Which JavaScript framework do you want? Angular 2
? Which module management do you want? Webpack with NPM
? Which CSS preprocessor do you want? SASS
? Which JS preprocessor do you want? TypeScript
```

I've modified the `hello.ts` to extract the html and I've created `hello.html` file, like this:

```
@Component({
  selector: 'hello-app',
  templateUrl: './app/hello.html'
})
```

So my new application structure is:

```
| src
|- app
|-- -- hello.html
|-- -- hello.spec.ts
|-- -- hello.ts
|-- index.ts
|-- index.html
|-- etc ..
```

No problem when I use `gulp serve` but when I try to generate a new distribution with `gulp serve:dist` the sub asset was not included in the dist folder.

```
Uncaught (in promise) Failed to load app/hello.html
```

Do you have a workaround for this problem ?

Thanks in advance.

**Comment by wojciechpolak** [comment]
I checked this also and it seems this case is with Webpack here. BTW, in my case, `templateUrl` doesn't work for me with JSPM while doing both `gulp serve` and  `gulp serve:dist`. The path is not resolved correctly.

**Comment by Swiip** [comment]
Handling non inline template in ng2 is a basic need ok but is not ready yet. There is still webpack / systemjs / browsersync configuration to adjust.

**Comment by Swiip** [comment]
Bundling ng2 template goes farther. We have to handle static template compilation to have the best of ng2.

**Comment by zckrs** [comment]
Released in 0.6.0
